src/OrbitQt/FilterPanelWidget.ui

```diff
@@ -38,7 +38,7 @@
       <string>FilterFunctions</string>
      </property>
      <property name="placeholderText">
-      <string>Filter Functions</string>
+      <string>Filter</string>
```

Comment:
FIlter seems to be pretty vague. I would still maintain Functions name or maybe Filter Scope? What do you think?

Reply:
Good point. The user doesn't have to know what exactly we call scope (and we call "scope" whatever gets issued equal `scope_id`). What if we call it "Filter Name"? Accordingly, we would re-name the "Function" bar under Live tab to "Name". As Live tab doesn't only have functions these days and this would be more coherent with the Sampling tab.